cli/azd/internal/cmd/provision.go

```diff
@@ -344,19 +344,30 @@ func (p *ProvisionAction) Run(ctx context.Context) (*actions.ActionResult, error
 		}
 	}
 
+	followUpLinks := []string{
+		getResourceGroupFollowUp(
+			ctx,
+			p.formatter,
+			p.portalUrlBase,
+			p.projectConfig,
+			p.resourceManager,
+			p.env,
+			false,
+		),
+	}
+
+	var additionalLinks map[string]*output.Link
+
+	if ok, err := p.env.Config.GetSection("provision.links", &additionalLinks); ok && err == nil {
```

Comment:
It feels a little strange to me to be smuggling this information via the config.  I feel like I'd almost rather introduce some method on the `ServiceTarget` interface that we walk over here (since we should have access to the `azure.yaml`) and asks each one what sort of links would you like to add?

Reply:
Yes, I am fine with doing that but we would need to introduce this method on all the service target implementations or create a base with some composition.  Happy to do this but didn't want to bloat this PR more.  

You good with taking this up as a fast follow and I can just remove this bit for the time being?

Reply:
Sure - works for me.

Reply:
> Happy to do this but didn't want to bloat this PR more.

BTW, thank you for taking this position.  This PR has been much easier review because there's not much ancillary stuff going on.  In the past I've struggled sometimes with the larger feature items because we have a tendency to couple larger refactorings with net new work and it has been hard for me to see the forest for the trees.  I didn't have that problem here and I think it's due to you holding the line on things like this.  As a reviewer, I really do appreciate it.